content/en/community/hackathons/sessions/baby-steps/content/Setup_Helloworld.md

```diff
@@ -0,0 +1,345 @@
+[...]
+
+$ ls -F
+[...]  Makefile  Makefile.uk  README.md  main.c  run-qemu-x86_64.sh* [...]
+```
+
+Now, while inside the `helloworld/` directory, we create the `.unikraft/` folder, which will store the Unikraft core kernel from this [repository](https://github.com/unikraft/unikraft), along with the `libs/` directory which should store all of the external libraries the applications depends on, cloned from the `lib-` repos of the [Unikraft organization](https://github.com/unikraft/).
+Since the `helloworld` application is fairly simple, it does not use any external library, so we can leave the `libs/` directory empty.
+
+```bash
+$ mkdir .unikraft
+
+$ cd .unikraft/
+
+$ git clone https://github.com/unikraft/unikraft unikraft
+
+$ mkdir libs
+```
+
+### Configuration
+
+Now that we have our setup ready, we can head back to the `helloworld/` directory and start configuring our application!
+
+```bash
+$ cd ../
+
+$ make menuconfig
+```
+
+This will open a menu-driven user interface in which you can choose the target architecture, virtualization platform, and configuration options for both internal and external libraries of the resulting unikernel image.
+
+We are met with the following configuration menu. Let's pick the x86 architecture:
+
+![arch selection menu](/community/hackathons/sessions/baby-steps/images/menuconfig_select_arch.png)
+
+![arch selection menu2](/community/hackathons/sessions/baby-steps/images/menuconfig_select_arch2.png)
+
+![arch selection menu3](/community/hackathons/sessions/baby-steps/images/menuconfig_select_arch3.png)
+
+Now, press `Exit` (or hit the `Esc` key twice) until you return to the initial menu.
+
+We have now set our desired architecture, let's now proceed with the platform.
+We will choose only the `KVM` platform (press the `Y` key to select it), which is supported by QEMU, the program we will use to launch our virtual machine after compilation:
+
+![plat selection menu](/community/hackathons/sessions/baby-steps/images/menuconfig_select_plat.png)
+
+![plat selection menu2](/community/hackathons/sessions/baby-steps/images/menuconfig_select_plat2.png)
+
+For the moment, this is sufficient. Press the `Save` button and exit the configuration menu by repeatedly selecting `Exit`.
+
+### Building
+We can now build our application. We can do this by simply invoking `make`:
+
+```bash
+$ make -j$(nproc)
+```
+
+This command invokes the build system by compiling the application on `$(nproc)` different threads, for shorter build times.
+The compilation process includes all of the Unikraft core internal libraries selected by default for this app, along with the application source files (just `main.c` for `helloworld`).
+The process should end with the following output:
+
+```bash
+[...]
+  LD      helloworld_qemu-x86_64.dbg
+  UKBI    helloworld_qemu-x86_64.dbg.bootinfo
+  SCSTRIP helloworld_qemu-x86_64
+  GZ      helloworld_qemu-x86_64.gz
+make[1]: Leaving directory '/home/X/workdir/helloworld/.unikraft/unikraft'
```

Comment:
Maybe briefly explain the output of `make` a little. Tell them that they can see the generated file and the operation that was done on it.